### DISCUSSION TITLE
fzf & ripogrep & vim

### DISCUSSION THREAD

**Discussion by Masber** [BurntSushi/ripgrep#2907] [Q&A] [answered]
Dear ripgrep community,

I am learning vim and trying to configure vim with fzf and repgrep, for some reason the fzf preview panel showing ripgrep result file content is not taking my vim colorscheme.

Is there a way to tell ripgrep to use my vim color scheme?

thank you very much

**Comment by BurntSushi** [comment, marked as answer]
The only way to control ripgrep colors is with the `--colors` flag. ripgrep doesn't automatically pick up any colorschemes.
